components/table/table-col-sort-button.js

```diff
@@ -37,30 +37,47 @@ export class TableColSortButton extends FocusMixin(LitElement) {
 				align-items: center;
 				background-color: transparent;
 				border: none;
+				box-sizing: border-box;
 				color: inherit;
 				cursor: pointer;
 				display: inline-flex;
 				font-family: inherit;
 				font-size: inherit;
+				height: var(--d2l-sortable-button-height);
 				letter-spacing: inherit;
+				line-height: 0.9rem;
 				margin: 0;
-				padding: 0;
+				padding: 0.75rem;
```

Comment:
I would still recommend using `--d2l-table-cell-padding` here. Since we want the padding of cells to now all be consistent, then if it gets overridden this location also stays consistent.